Bump old dependencies so it is possible to run tests with -Z minimal-versions

compiler_builtins bumped due to build failure in compiler_builtins.


Index: object/Cargo.toml
===================================================================
--- object.orig/Cargo.toml
+++ object/Cargo.toml
@@ -35,7 +35,7 @@ features = ["doc"]
 
 
 [dependencies.compiler_builtins]
-version = "0.1.2"
+version = "0.1.70"
 optional = true
 
 [dependencies.crc32fast]
